packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -104,7 +104,13 @@ export class TreeItem extends FoundationElement {
 
     public handleClick = (e: MouseEvent): void | boolean => {
         if (!e.defaultPrevented) {
-            this.handleSelected(e);
+            const target = e.composedPath();
```

Comment:
minor nit but can we call this `targets`? it's a collection, not a singular target